packages/auth-client/src/controllers/engine.ts

```diff
@@ -191,6 +194,13 @@ export class AuthEngine extends IAuthEngine {
     return pendingRequests;
   };
 
+  public formatMessage: IAuthEngine["formatMessage"] = (
+    payload: AuthEngineTypes.CacaoPayload,
+    iss: string,
+  ) => {
+    return this.constructEip4361Message(payload, iss);
+  };
```

Comment:
Why are we making a new function that does nothing but send the same params in the same shape to `constructEip4361Message`? Wouldn't it make more sense to just rename the `constructEip4361Message`? Unless we plan on having different message formats, and even then they'd still probably only take `payload` and `iss`

Reply:
good point :+1: